packages/okta-adapter/src/filters/group_members.ts

```diff
@@ -107,6 +237,48 @@ const groupMembersFilter: FilterCreator = ({ config, paginator }) => ({
 
     groupMembershipInstances.forEach(instance => elements.push(instance))
   },
+  deploy: async changes => {
+    const [relevantChanges, leftoverChanges] = _.partition(
+      changes,
+      change =>
+        isInstanceChange(change) &&
+        isAdditionOrModificationChange(change) &&
+        getChangeData(change).elemID.typeName === GROUP_MEMBERSHIP_TYPE_NAME,
+    )
+
+    const { includeGroupMemberships } = config[FETCH_CONFIG]
+    if (!includeGroupMemberships && relevantChanges.length > 0) {
+      log.error('group memberships flag is disabled')
+      return {
+        leftoverChanges,
+        deployResult: {
+          appliedChanges: [],
+          errors: relevantChanges.map(change => ({
+            elemID: getChangeData(change).elemID,
+            severity: 'Error',
+            message:
+              'Group membership is disabled. To apply this change, change fetch.includeGroupMemberships flag to “true” in your Okta environment configuration.',
+          })),
+        },
+      }
+    }
+    const deployResult = await Promise.all(
+      relevantChanges.map(async change =>
+        deployGroupMembershipChange(
+          change as AdditionChange<InstanceElement> | ModificationChange<InstanceElement>,
+          client,
+        ),
+      ),
+    )
```

Comment:
very small nit since I personally doesn't like to use explicit type casting ( I see that you already checked it in line 245 so its really up to you) :
```suggestion
    const deployResult = await Promise.all(
      relevantChanges.filter(isAdditionOrModificationChange).filter(isInstanceChange)
      .map(async change =>
        deployGroupMembershipChange(
          change,
          client,
        ),
      ),
    )